src/scripts/eos-analysis_TEST

```diff
@@ -10,15 +10,15 @@ echo using PYTHON=${PYTHON} >&2
 echo using PYTHONPATH=${PYTHONPATH} >&2
 
 ###########################################
-## Run the Analysis's steps automaticall ##
+## Run the Analysis's steps automatically ##
 ###########################################
 echo running analysis steps test \(CKM+FF.find-mode\)... >&2
 $PYTHON ${SOURCE_DIR}/eos-analysis \
     run \
     CKM+FF.find-mode \
     -f ${SOURCE_DIR}/eos-analysis_TEST.d/analysis.yaml
 echo ... success >&2
-echo running analysis steps test \(CKM+ff.sample\)... >&2
+echo running analysis steps test \(CKM+FF.sample\)... >&2
```

Comment:
how did that happen?

Reply:
A certain DvD mistyped when he wrote this line a few weeks ago? It's just the echo message saying what's happening so no actual issue